staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -35,18 +36,20 @@ var (
 )
 
 // PanicHandlers is a list of functions which will be invoked when a panic happens.
-var PanicHandlers = []func(interface{}){logPanic}
+var PanicHandlers = []func(context.Context, interface{}){logPanic}
```

Comment:
How many projects set this manually? My expecation is nearly nobody. Can you confirm?

Reply:
I find four places in controller-runtime, Cilium, Gardener: https://grep.app/search?q=utilruntime.PanicHandlers&filter[path][0]=pkg/

Reply:
This assumes that they use the `utilruntime` import name. Might be a bit more which don't use it like that.